vlasiator.cpp

```diff
@@ -494,17 +496,23 @@ int main(int argn,char* args[]) {
 
    phiprof::start("getFieldsFromFsGrid");
    volGrid.updateGhostCells();
-   getFieldsFromFsGrid(volGrid, BgBGrid, EGradPeGrid, technicalGrid, mpiGrid, cells);
+   getFieldsFromFsGrid(volGrid, BgBGrid, EGradPeGrid, dMomentsGrid, technicalGrid, mpiGrid, cells);
```

Comment:
update dMomentsGrid before getting values from there or it'll go bust as well, no?